### DISCUSSION TITLE
Square icon with rounded corner for macOS?

### DISCUSSION THREAD

**Discussion by sharpordie** [TiagoSilvaPereira/vemto-issues#364] [Ideas & Suggestions]
It would be great to have a nice macOS style icon for macOS.
I am currently using this smartgit icon from the macosicons website, but it's weird.

![](https://user-images.githubusercontent.com/72373746/168786269-f6291db7-84d5-46cf-9dff-c55fe39896bc.png)

It is not of the utmost importance, just details. But details are important. ;)

EDIT: Oh I just saw that it would be as well to titlecase the word vemto too.

**Comment by TiagoSilvaPereira** [comment]
@sharpordie thank you for your suggestion

**Comment by TiagoSilvaPereira** [comment]
Moving as suggestion for Vemto 2
